Tests/LibWeb/Text/expected/css/getComputedStyle-print-all.txt

```diff
@@ -102,7 +102,7 @@ list-style-position: outside
 list-style-type: disc
 margin-block-end: 8px
 margin-block-start: 8px
-margin-bottom: 8px
+margin-bottom: 16px
```

Comment:
This appears to be incorrect, but I'm not sure whether this is due to something I've done, or some issue with margin calculation elsewhere.

Reply:
I've decided to scale back this PR to only change how width values are resolved, since the above is clearly incorrect. Given that all the WPT tests mentioned in the PR description still pass when only the width calculation is changed, it seems like a worthwhile change on its own.